exporter/collector/config.go

```diff
@@ -28,23 +28,33 @@ const (
 type Config struct {
 	ProjectID string `mapstructure:"project"`
 	UserAgent string `mapstructure:"user_agent"`
```

Comment:
Should we move this into the ClientConfig as well?

Reply:
Good question.  I initially moved it into different ones for trace/metrics, since it is part of the config for clients.  I haven't found a concrete definition for what should be put in user agent though... It seems to usually be defined as the "software" that is making the request, and I consider the exporter to be the same piece of "software", but we could definitely split it as well.